components/core/SearchModal.js

```diff
@@ -83,7 +84,12 @@ const UserEntry = ({ user }) => {
   return (
     <div css={STYLES_ENTRY}>
       <div css={STYLES_ENTRY_CONTAINER}>
-        <div style={{ backgroundImage: `url(${user.data.photo})` }} css={STYLES_PROFILE_PREVIEW} />
+        <div css={STYLES_PROFILE_PREVIEW}>
+          <ProfilePhoto 
+            user={this.props.viewer}
```

Comment:
I think you mean to pass in the user here not this.props.viewer
![image](https://user-images.githubusercontent.com/33686587/126019032-8d9051df-0bdf-46df-8b81-b4252c0cd432.png)